pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/ColumnIndexCreationInfo.java

```diff
@@ -37,11 +37,12 @@
   private final PartitionFunction partitionFunction;
   private final List<IntRange> partitionRanges;
   private final Object defaultNullValue;
+  private int numPartitions;
```

Comment:
For offline it is not necessarily kafka partitions. I think numPartitions is fine.